docs/appendices/backend-writers-guide.rst

```diff
@@ -793,7 +795,7 @@ other update/remove functionality at a later stage.
   Please keep in mind that the zone is not empty because
   ``startTransaction()`` was called differently.
 
-.. cpp:function:: virtual bool DNSBackend::listSubZone(const DNSName &name, int domain_id)
+.. cpp:function:: virtual bool DNSBackend::listSubZone(const DNSName &name, domainid_t domain_id)
```

Comment:
apparently missed before: the first arg is a `ZoneName` now. Goes for some other entries too.